Extensions/PlayerAuthentication/playerauthenticationtools.ts

```diff
@@ -284,7 +327,7 @@ namespace gdjs {
      * Reads the event sent by the authentication window and
      * display the appropriate banner.
      */
-    const receiveMessageFromAuthenticationWindow = function (
+    const receiveMessageFromLiluo = function (
```

Comment:
I kind of liked "Authentification window" - but I see that it's now not only the window but the parent.
Could be `receiveAuthenticationMessage`?